plugin/web/extensions/location/modals/googleAnalyticsModal/index.tsx

```diff
@@ -0,0 +1,60 @@
+import useHook from "@web/extensions/location/hook";
+import { Typography } from "@web/sharedComponents";
+import { styled } from "@web/theme";
+
+import CommonModalWrapper from "../commonModalWrapper";
+
+const GoogleAnalyticstModal: React.FC = () => {
+  const { handlegoogleModalChange } = useHook();
+
+  const { Link } = Typography;
+
+  return (
+    <CommonModalWrapper
+      title="Google Analytics の利用について"
+      onModalChange={handlegoogleModalChange}>
+      <>
```

Comment:
Again, I don't think many of these fragments are needed. You will only need it if you want to create a section with a few elements that aren't wrapped together.

Reply:
386626f0ec753cdcae932611366d0e17a73470a4

Reply:
the fragments removed as discussed in previous comments